libpoolprotocols/stratum/EthStratumClient.cpp

```diff
@@ -118,6 +118,11 @@ void EthStratumClient::connect()
 			SSL_CTX_set_cert_store(ctx.native_handle(), store);
 #else
 			ctx.set_default_verify_paths();
+			try {
```

Comment:
This will override potential use of SSL_CERT_FILE env. var. I had tried using SSL_CERT_FILE and SSL_CERT_FILE not specifying the full path in SSL_CERT_FILE. That didn't work!
Specifying the full path in SSL_CERT_FILE works fine. So I would lean towards just instructing Linux users to use SSL_CERT_FILE as you have in the help.

Reply:
well, i could check in code if the env var is set and then don't manually load the file.

Reply:
Sure, that would be advisable if the Ubuntu default doesn't apply.

Reply:
Don't think there's any point in calling set_default_verify_paths anymore.

How something like about:

                        char *certPath = getenv("SSL_CERT_FILE");
                        try {
                                ctx.load_verify_file(certPath ? certPath : "/etc/ssl/certs/ca-certificates.crt");
                        }
                        catch (...) {
                        }

Reply:
Ok, will test the code and then add it this way. I even can catch a load error there and show a warning there too.

Reply:
Yes. The warning was good.

Reply:
Pushed your recommendation + warning message on failure